### PR TITLE
Add python decouple to the project.

### PR DESCRIPTION
**Pull request by akatsoulas**

### CODE REVIEW

**Comment by akatsoulas** [comment]
fixes #105

File: moderator/settings.py [hunk 1]
```diff
@@ -146,66 +112,77 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
...
+ALLOWED_HOSTS = config('ALLOWED_HOSTS', default='http://127.0.0.1:8000', cast=Csv())
+
+# Instruct session-csrf to always produce tokens for anonymous users
+ANON_ALWAYS = config('ANON_ALWAYS', default=True, cast=bool)
 
 # Create account for new users.
-BROWSERID_CREATE_USER = True
+BROWSERID_CREATE_USER = config('BROWSERID_CREATE_USER', default=True, cast=bool)
 BROWSERID_VERIFY_CLASS = 'moderator.moderate.views.BrowserIDVerify'
+BROWSERID_AUDIENCES = config('BROWSERID_AUDIENCES', default='http://127.0.0.1:8000')
```
Comment (johngian) [inline moderator/settings.py]: I think that this should be a list, not just a string.

File: moderator/settings.py [hunk 2]
```diff
@@ -146,66 +112,77 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
...
+
+# Internationalization
+TIME_ZONE = config('TIME_ZONE', default='UTC')
+USE_I18N = config('USE_I18N', default=False, cast=bool)
+USE_L10N = config('USE_L10N', default=False, cast=bool)
+USE_TZ = config('USE_TZ', default=True, cast=bool)
+
+SECRET_KEY = config('SECRET_KEY')
+DEBUG = config('DEBUG', default=False, cast=bool)
+SITE_URL = config('SITE_URL', default='http://127.0.0.1:8000')
```
Comment (johngian) [inline moderator/settings.py]: I would prefer to avoid having localhost as default, instead we can define it in .env

File: moderator/settings.py [hunk 3]
```diff
@@ -146,66 +112,77 @@ def COMPRESS_JINJA2_GET_ENVIRONMENT():
...
+ALLOWED_HOSTS = config('ALLOWED_HOSTS', default='http://127.0.0.1:8000', cast=Csv())
```
Comment (johngian) [inline moderator/settings.py]: Same here.

**Comment by johngian** [comment]
HMAC_KEYS should also be able to be configured in env.

File: moderator/settings.py [hunk 4]
```diff
@@ -2,6 +2,8 @@
 # repo. If you need to override a setting locally, use local.py
 import os
 
+from decouple import Csv, config
+import dj_database_url
```
Comment (johngian) [inline moderator/settings.py]: Please fix order of imports here.

File: moderator/settings.py (hunk 1 above)
Comment (johngian) [inline moderator/settings.py]: Also let's avoid defining hardcoded IPs

File: env-dist [hunk 5]
```diff
@@ -0,0 +1,8 @@
+SECRET_KEY='some secret'
+MOZILLIANS_API_BASE="https://mozillians.org/api/v1/users/"
```
Comment (johngian) [inline env-dist]: Please use single quotes.

**Comment by johngian** [comment]
r+

**Pull request merged**